css/styles.scss

```diff
@@ -107,3 +107,31 @@ html, body {
     }
   }
 }
+
+.navbar-fixed{
```

Comment:
indentation before the bracket is a little off, please maintain consistency with existing code.